values-schema.yaml

```diff
@@ -16,14 +14,18 @@ definitions:
         properties:
           critical:
             $ref: '#/definitions/email'
-            description: One or more email addresses (comma separated) for critical events.
+            description: One or more email addresses (comma separated) for critical
```

Comment:
There is just sorting & formatting here, I don't know if it's useful/outside of the scope but it reads a little easier.

Reply:
PS I didn't do this by hand of course